Terminal.Gui/Drawing/Thickness.cs

```diff
@@ -273,7 +273,7 @@ public Rect GetInside (Rect rect)
         int width = Math.Max (0, rect.Size.Width - Horizontal);
         int height = Math.Max (0, rect.Size.Height - Vertical);
 
-        return new Rect (new Point (x, y), new Size (width, height));
+        return new (x, y, width, height);
```

Comment:
Stack churn of creating these intermediate structs is unnecessary. Just make it via this constructor.